builders/get-started/moonbeam.md

```diff
@@ -34,7 +34,7 @@ If you do not have MetaMask installed, or would like to follow a tutorial to get
 If you want to connect MetaMask by providing the network information, you can use the following data:
 
  - Network Name: `Moonbeam`
- - RPC URL: `{{ networks.moonbeam.rpc_url }}`
+ - RPC URL: `{{ networks.moonbeam.public_rpc_url }}`
```

Comment:
I would remove the Public Endpoint here as well. In any case, I would leave it in the "Interact with Metamask" page, so at least is a bit more "buried". They can use the buttons to connect Metamask to Moonbeam

Reply:
so what URL should I put here?

Reply:
Or are you saying to completely scrap the entire network information? because I think it would be weird to have everything you need to connect to metamask except for the url